data-access-services/pom.xml

```diff
@@ -49,6 +49,15 @@
                         </excludes>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-checkstyle-plugin</artifactId>
+                    <version>3.1.2</version>
```

Comment:
Maybe declare the version in the properties above, so they can be viewed in one place at the top of the file?

Reply:
sure, updated.